js/components/wysiwyg.js

```diff
@@ -42,11 +42,9 @@ Fliplet.FormBuilder.field('wysiwyg', {
     onReset: function () {
       if (this.editor) {
         try {
-          return this.editor.setContent('');
+          return this.editor.setContent(this.value);
```

Comment:
isn't this the current value, not the initial?

Reply:
No, at this point this.value was already reassigned to the initial value